osler/inventory/admin.py

```diff
@@ -9,3 +10,5 @@
 admin.site.register(models.MeasuringUnit)
 
 admin.site.register(models.Manufacturer)
```

Comment:
I like to do these in a loop (`for model in [model.Drug, models.DrugCategory, ...]`). Makes it easy to add more later without copy/pasting.